### PR TITLE
Catalog Grid

### PR DESCRIPTION
**Pull request by 0xNazarii**
- [DEMO LINK](https://nzrii-kysel.github.io/layout_catalog_grid/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -15,43 +15,43 @@
...
+            <a href="#samsung" class="nav__link">Samsung</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Smartphones</a>
+            <a href="#smartphones" class="nav__link">Smartphones</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Laptops & computers</a>
+            <a href="#laptops" class="nav__link">Laptops & computers</a>
           </li>
```
Comment (Sergyurch) [inline src/index.html:37]: Insert empty lines between li tags.

File: src/styles/utils/normalize.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
```
Comment (Sergyurch) [inline src/styles/utils/normalize.scss:8]: I would rather set font-family for page block, not in normalize file.

**Review by Sergyurch — APPROVED** [review]
Very nice.

File: src/styles/utils/normalize.scss (hunk 2 above)
Comment (nzrii-kysel) [reply src/styles/utils/normalize.scss]: thanks, it does seem like a better practice

**Review by ArturErmolenko — APPROVED** [review]
GJ 👍

**Pull request closed without merging**